wom/src/main/scala/wdl4s/wdl/WdlWorkflow.scala

```diff
@@ -51,7 +51,7 @@ object WdlWorkflow {
 
   def buildWomGraph(wdlWorkflow: WdlWorkflow): Graph = {
     val graphNodes = wdlWorkflow.calls.foldLeft(Set.empty[GraphNode])({
-      case (currentNodes, call) => currentNodes ++ call.womGraphInputNodes + call.womCallNode
+      case (currentNodes, call) => currentNodes ++ call.womGraphInputNodes.getOrElse(throw new Exception()) + call.womCallNode.getOrElse(throw new Exception())
```

Comment:
Related to the point @mcovarr made, if these are Exceptions for no other reason than because `ErrorOr` (or `Try`) expects one, please use something else which just takes the message (e.g. error oriented ADT or an alternative Validated type)